pkg/templates/templates_generated.go

```diff
@@ -3123,6 +3123,9 @@ write_files:
   owner: root
   content: |
     {
+      {{- if HasDataDir . -}}
```

Comment:
seems the last dot is not necessary?

Reply:
yep, fixed this too. Both bad copy-paste